quickshow/src/cellrenderer.cpp

```diff
@@ -3,6 +3,9 @@
 #include <qf/core/log.h>
 #include <qf/core/utils.h>
 
+#include "application.h"
```

Comment:
Most specific includes must be be first, at the beginning in this case.

Reply:
fixed